feat: allow different bot name to auto-merge (#233)
ridedott/dependabot-auto-merge-action: README.md

        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

### Enable auto-merge for a different bot

> You may have another bot that also raises PR against your repo and you want to

Reviewer: ```suggestion
You may have another bot that also creates PRs against your repository and you want to
```

> auto-merge. By default, this github action will assume the bot is dependabot.

Reviewer: ```suggestion
automatically merge those. By default, this GitHub Action assumes the bot is [`dependabot`](https://dependabot.com/).
```

> You can override the bot name by setting up the value `GITHUB_LOGIN`:

Reviewer: ```suggestion
You can override the bot name by changing the value of `GITHUB_LOGIN` parameter:
```

```yaml
steps:
  - name: Auto merge
    uses: ridedott/dependabot-auto-merge-action@master
    with:
      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}

>       GITHUB_LOGIN: my-awesome-bot-r2d2

Reviewer: Sort :)

Author: fixed with https://github.com/ridedott/dependabot-auto-merge-action/pull/233/commits/3e01c991795106436edb97fcce1139496c4eb0c9
